core/che-core-api-jdbc/src/main/java/org/eclipse/che/api/core/jdbc/jpa/eclipselink/AbstractDescriptorEventListener.java

```diff
@@ -0,0 +1,71 @@
+package org.eclipse.che.api.core.jdbc.jpa.eclipselink;
+
+import org.eclipse.persistence.descriptors.DescriptorEvent;
+import org.eclipse.persistence.descriptors.DescriptorEventListener;
+
+import java.util.Vector;
+
+/**
+ * A skeletal implementation of {@link DescriptorEventListener}.
+ *
+ * @author Yevhenii Voevodin
+ */
+public abstract class AbstractDescriptorEventListener implements DescriptorEventListener {
```

Comment:
I'd prefer to let the current name, afaik awt provides some skeletal implementations called adapters, but from the _Effective Java 2nd ed_:

> While interfaces are not permitted to contain method implementations, using
interfaces to define types does not prevent you from providing implementation
assistance to programmers. You can combine the virtues of interfaces and
abstract classes by providing an abstract skeletal implementation class to go
with each nontrivial interface that you export. The interface still defines the
type, but the skeletal implementation takes all of the work out of implementing it.
By convention, skeletal implementations are called AbstractInterface, where
Interface is the name of the interface they implement. For example, the Collec-
tions Framework provides a skeletal implementation to go along with each main
collection interface: AbstractCollection, AbstractSet, AbstractList, and
AbstractMap. Arguably it would have made sense to call them SkeletalCollec-
tion, SkeletalSet, SkeletalList, and SkeletalMap, but the Abstract conven-
tion is now firmly established.